charts/feature-toggle-api/templates/bin/_dns-register.tpl

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
```

Comment:
i think you want to add a gitignore entry for *.tgz since the postgres tar has crept in again